app/bettermail.go

```diff
@@ -127,6 +135,72 @@ func handlePushPayload(payload PushPayload, c appengine.Context) (*mail.Message,
 	return message, nil
 }
 
+func fetchCommit(payload CommitCommentPayload, commit *ApiCommit, c appengine.Context) error {
+	commitSHA := *payload.Comment.CommitID
+	url := strings.Replace(*payload.Repo.CommitsURL, "{/sha}", "/" + commitSHA, 1)
+
+	client := urlfetch.Client(c)
+	resp, err := client.Get(url)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	return json.NewDecoder(resp.Body).Decode(commit)
+}
+
+func handleCommitCommentPayload(payload CommitCommentPayload, c appengine.Context) (*mail.Message, error) {
+	var commit ApiCommit
+	title := ""
+	err := fetchCommit(payload, &commit, c)
+	if err != nil {
+		c.Infof("Error fetching commit = %v\n", err)
+	} else {
+		title, _ = getTitleAndMessageFromCommitMessage(*commit.Commit.Message)
+	}
+
+	// TODO: allow location to be customized
+	location, _ := time.LoadLocation("America/Los_Angeles")
+	updatedDate := payload.Comment.UpdatedAt.In(location)
+
+	commitShortSHA := (*payload.Comment.CommitID)[:7]
+
+	var data = map[string]interface{}{
+		"Payload":                  payload,
+		"Comment":                  payload.Comment,
+		"Sender":                   payload.Sender,
+		"Repo":                     payload.Repo,
+		"ShortSHA":                 commitShortSHA,
+		"Body":                     *payload.Comment.Body,
```

Comment:
Would be nice to do markdown rendering of the comment body. We do this for commit bodies already, see `newDisplayCommit` for an example.